lib/utils/permissionList.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+const R = require('ramda');
+const scriptData = require('./scriptData');
+
+function getPermissionMappings (type) {
+  const MAPPINGS = {
+    permission: [1],
+    description: [1],
```

Comment:
if both have the same value, why are they separate fields?

Reply:
@facundoolano 
Google does not provide "permission" field anymore, only description.
So, for backward compatibility I used the same field for both permission and description.
I can comment on this too.

You can check the discussion and response return [here](https://github.com/facundoolano/google-play-scraper/pull/337#issuecomment-529534740) 

May I suggest a file of changeslog?
Where we can add what was changed between versions, instead of comment each file?

Reply:
A changelog would certainly be nice, I just never got around to it. 

I don't think we need to keep things around for backward compatibility. Being a scraper we have to break things all the time anyway to keep up with the page changes.